src/components/header/PrimaryNav/PrimaryNav.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import classnames from 'classnames'
+
+import { NavButton } from '../NavButton/NavButton'
+import { List } from '../List/List'
+
+// assets
+import closeImg from 'uswds/src/img/close.svg'
+
+type PrimaryNavProps = {
+  items: React.ReactNode[]
+  mobileExpanded?: boolean
+}
+
+export const PrimaryNav = (
+  props: PrimaryNavProps & React.HTMLAttributes<HTMLButtonElement>
```

Comment:
Since this component is rendering a `<nav>` element it should be `React.HTMLAttributes<HTMLElement>` (see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)

Reply:
ooops, typo